tools/smoke_tests.py

```diff
@@ -82,15 +82,18 @@ def main():
 
     # Test modeling.
     logger.info('Testing Civis-ML...')
-    mp = civis.ml.ModelPipeline(
-        model="sparse_logistic",
-        dependent_variable="type",
-        primary_key="index",
-        client=client
-    )
-    result = mp.train(
-        table_name="datascience.iris", database_name=database).result()
-    assert result['state'] == 'succeeded'
+    for civisml_version in (None, 'v2.2'):  # None = latest production version
```

Comment:
Just to clarify, specifying `v.2.2` is so that you'll be testing both the production and versioned alias? This test won't need to be updated if a new template is released, right?

Reply:
> specifying v.2.2 is so that you'll be testing both the production and versioned alias?

Correct.

> This test won't need to be updated if a new template is released, right?

Correct.